### PR TITLE
ConnectionSetupPayload: expose resume flag & token

### PR DESCRIPTION
**Pull request by mostroverkhov**
Relates to #665

### CODE REVIEW

**Comment by mostroverkhov** [comment]
looks like `develop` build (openjdk) is not green - there are failing tests `testThatSetupWithSpecifiedFrameSizeButLowerThanWsDefaultShouldSetToWsDefault`

File: rsocket-core/src/main/java/io/rsocket/ConnectionSetupPayload.java [hunk 1]
```diff
@@ -101,6 +106,18 @@ public boolean willClientHonorLease() {
       return SetupFrameFlyweight.honorLease(setupFrame);
     }
 
+    @Override
+    public boolean isResumeEnabled() {
+      return SetupFrameFlyweight.resumeEnabled(setupFrame);
+    }
+
+    @Override
+    public ByteBuf resumeToken() {
```
Comment (yschimke) [inline rsocket-core/src/main/java/io/rsocket/ConnectionSetupPayload.java]: Should this be null instead?  For things like metadata, null means not present.

Comment (mostroverkhov) [reply rsocket-core/src/main/java/io/rsocket/ConnectionSetupPayload.java]: resume token cant be empty - there is no need to distinguish with null.  Looks like this change can be reduced to one token returning method - and empty would mean non-resumable

Comment (mostroverkhov) [reply rsocket-core/src/main/java/io/rsocket/ConnectionSetupPayload.java]: @yschimke checking spec `Unsigned 16-bit integer of Resume Identification Token Length in bytes. (Not present if R flag is not set)`, having both `isResumeEnabled` and nullable `resumeToken` seems best option.

**Pull request merged**